app/components/provider_interface/interview_form_component.rb

```diff
@@ -0,0 +1,17 @@
+module ProviderInterface
+  class InterviewFormComponent < ViewComponent::Base
+    attr_reader :application_choice, :form_model, :form_url, :form_heading
+
+    def initialize(application_choice:, form_model:, form_url:, form_heading:)
+      @application_choice = application_choice
+      @form_model = form_model
+      @form_url = form_url
+      @form_heading = form_heading
+    end
+
+    def example_date
+      valid_future_date_for_form = [application_choice.reject_by_default_at, 1.day.from_now].min
```

Comment:
There was a question about whether it is useful to display today’s date in the case the application is going to be rejected that day.

Given that it is the only date that would be accepted by the form, it feels like the only choice here